CI is currently failing?

https://github.com/litex-hub/pythondata-auto/runs/3808023659?check_suite_focus=true
```
Cloning into bare repository 'srcs/pythondata-software-picolibc'...

picolibc Config
picolibc
  picolibc 0.0.post21344 (0, 0, 21344)
  Tools: 0.0.post105 (0, 0, 105)
   Data: 0.0.post21239 (0, 0, 21239)
  Updating repo  litex-hub/pythondata-software-picolibc
  404 {"message": "Not Found", "documentation_url": "https://docs.github.com/rest/reference/repos#update-a-repository"}
  Cloning into 'repos/pythondata-software-picolibc'...
  Submodule 'pythondata_software_picolibc/data' (https://github.com/picolibc/picolibc) registered for path 'pythondata_software_picolibc/data'
  Cloning into '/home/runner/work/pythondata-auto/pythondata-auto/repos/pythondata-software-picolibc/pythondata_software_picolibc/data'...
  Submodule path 'pythondata_software_picolibc/data': checked out '53567cf82b92f1bd777ed57111232a110666b4df'
  
  Updating: pythondata-software-picolibc
  ---------------------------------------------------------------------------
    Updating repos/pythondata-software-picolibc                           from /home/runner/work/pythondata-auto/pythondata-auto/templates
    Creating repos/pythondata-software-picolibc/pythondata_software_picolibc from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__
    Creating repos/pythondata-software-picolibc/.github                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github
     Copying repos/pythondata-software-picolibc/Makefile                  from /home/runner/work/pythondata-auto/pythondata-auto/templates/Makefile
   Rendering repos/pythondata-software-picolibc/MANIFEST.in               from /home/runner/work/pythondata-auto/pythondata-auto/templates/MANIFEST.in.jinja
   Rendering repos/pythondata-software-picolibc/README.md                 from /home/runner/work/pythondata-auto/pythondata-auto/templates/README.md.jinja
     Copying repos/pythondata-software-picolibc/requirements.txt          from /home/runner/work/pythondata-auto/pythondata-auto/templates/requirements.txt
   Rendering repos/pythondata-software-picolibc/setup.py                  from /home/runner/work/pythondata-auto/pythondata-auto/templates/setup.py.jinja
   Rendering repos/pythondata-software-picolibc/test.py                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/test.py.jinja
     Copying repos/pythondata-software-picolibc/.gitattributes            from /home/runner/work/pythondata-auto/pythondata-auto/templates/.gitattributes
     Copying repos/pythondata-software-picolibc/.gitignore                from /home/runner/work/pythondata-auto/pythondata-auto/templates/.gitignore
    Updating repos/pythondata-software-picolibc/pythondata_software_picolibc from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__
   Rendering repos/pythondata-software-picolibc/pythondata_software_picolibc/__init__.py from /home/runner/work/pythondata-auto/pythondata-auto/templates/__py__/__init__.py.jinja
    Updating repos/pythondata-software-picolibc/.github                   from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github
    Creating repos/pythondata-software-picolibc/.github/workflows         from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows
    Updating repos/pythondata-software-picolibc/.github/workflows         from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows
     Copying repos/pythondata-software-picolibc/.github/workflows/publish-to-pypi.yml from /home/runner/work/pythondata-auto/pythondata-auto/templates/.github/workflows/publish-to-pypi.yml
  ---------------------------------------------------------------------------
  git submodule update --remote --merge
  On branch master
  Your branch is up to date with 'origin/master'.
  
  nothing to commit, working tree clean
  Traceback (most recent call last):
    File "./update.py", line 587, in <module>
      sys.exit(main(sys.argv[0], sys.argv[1:]))
    File "./update.py", line 566, in main
      update(m)
    File "./update.py", line 405, in update
      subprocess_check_call(['git', 'commit', '-F', f.name], cwd=repo_dir)
    File "./update.py", line 27, in subprocess_check_call
      return subprocess.check_call(*args, **kw)
    File "/usr/lib/python3.6/subprocess.py", line 311, in check_call
      raise CalledProcessError(retcode, cmd)
  subprocess.CalledProcessError: Command '['git', 'commit', '-F', '/tmp/tmpus_azzns']' returned non-zero exit status 1.
  Error: Process completed with exit code 1.
```

--- a/update.py
+++ b/update.py
@@ -392,9 +392,12 @@ Updated using {tool_version} from https://github.com/litex-hub/litex-data-auto
             print(cmd)
             subprocess_check_call(cmd.split(), cwd=repo_dir)
             # submodule bump does not commit by itself
-            subprocess_check_call(['git', 'add', '.'], cwd=repo_dir)
-            with tempfile.NamedTemporaryFile() as f:
-                f.write("""\
+            tocommit = subprocess.check_output(
+                ['git', 'status', '--porcelain'], cwd=repo_dir).decode('utf-8')
+            if tocommit:
+                subprocess_check_call(['git', 'add', '.'], cwd=repo_dir)
+                with tempfile.NamedTemporaryFile() as f:
+                    f.write("""\
 Bump {dir} submodule to {data_git_hash}
 
 Updated using {tool_version} from https://github.com/litex-hub/litex-data-auto

--- a/update.py
+++ b/update.py
@@ -379,7 +379,6 @@ Updated data to {data_git_describe} based on {data_git_hash} from {src}.
 Updated using {tool_version} from https://github.com/litex-hub/litex-data-auto
 """.format(**module_data).encode('utf-8'))
             f.flush()
-            f.name
             subprocess_check_call(['git', 'commit', '-F', f.name], cwd=repo_dir)
 
     # Run the git subtree command
@@ -401,7 +400,6 @@ Bump {dir} submodule to {data_git_hash}
 Updated using {tool_version} from https://github.com/litex-hub/litex-data-auto
 """.format(**module_data).encode('utf-8'))
                 f.flush()
-                f.name
                 subprocess_check_call(['git', 'commit', '-F', f.name], cwd=repo_dir)
 
         else:

